Resources/Prototypes/Entities/Objects/Weapons/Guns/Shotguns/shotguns.yml

```diff
@@ -294,3 +294,29 @@
   components:
   - type: BallisticAmmoProvider
     proto: ShellShotgunImprovised
+
+- type: entity
+  parent: [BaseWeaponShotgun, BaseGunWieldable, BaseSyndicateContraband]
+  id: WeaponShotgunHushpup
+  name: Hushpup
+  description: An obscure modification of the Enforcer, fitted with an experimental muffler. Great for those with a strict moral code. Uses .50 shotgun shells.
+  components:
+  - type: Sprite
+    sprite: Objects/Weapons/Guns/Shotguns/hushpup.rsi
+  - type: Clothing
+    sprite: Objects/Weapons/Guns/Shotguns/hushpup.rsi
+  - type: Item
+    sprite: Objects/Weapons/Guns/Shotguns/hushpup.rsi
+    shape:
+    - 0,0,3,0 # smaller than BaseShotgun by 1 tile
+  - type: BallisticAmmoProvider
+    capacity: 4
+  - type: GunRequiresWield
+  - type: Gun
+    soundGunshot:
+      path: /Audio/Weapons/Guns/Gunshots/hushpup.ogg
+      params:
+        volume: -3
```

Comment:
```suggestion
        volume: -3
        rolloffFactor: 1.3
```

Due to how distinct the noise is and its volume (which is necessary for closerange), I'm opting to propose adding a rolloffFactor of 1.3

<img width="1689" height="602" alt="image" src="https://github.com/user-attachments/assets/8b774062-2097-4251-83c8-490b6533dbf1" />

In the screenshot below you can see the difference in audible distance, at 1 (default), 1.2, 1.3 and 1.5 from left to right.

1.5 leaves the listener right at the edge of the screen, so 1.3 seems like a decent compromise. If you include walls muffling the pitch, this is around the distance the cobra is *functionally* inaudible (though the cobra is technically still audible at the blue line).